src/Microsoft.SqlTools.ServiceLayer/TaskServices/Contracts/TaskInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.SqlTools.ServiceLayer.TaskServices.Contracts
+{
+    public enum TaskState
+    {
+        NotStarted = 0,
+        Running = 1,
+        Complete = 2
```

Comment:
This PR is just setting up placeholder for next step of work.  I think there are other completed states like failed, aborted, etc. that we will need to communicate.  We can refactor when we do the actual task management implementation in the next round of PRs.